tests/ops/qubit/test_attributes.py

```diff
@@ -471,3 +472,25 @@ def test_pcphase(self):
         expected_mat = np.array(mats)
         assert np.allclose(mat1, expected_mat)
         assert np.allclose(mat2, expected_mat)
+
+
+# pylint:disable=too-few-public-methods
+class TestHasUnitaryGenerator:
+    """Test that all operations in the ``has_unitary_generator`` attribute
+    actually have unitary generators."""
+
+    unitarily_generated_ops = sorted(list(has_unitary_generator))
```

Comment:
Again, why do we need to sort this list?

Reply:
Because the `Attribute` class extends `set`, this list would not have a consistent order without sorting - it would fail if parallel test execution takes place, as described above.